lib/codecov.js

```diff
@@ -5,7 +5,6 @@ var urlgrey = require('urlgrey')
 var jsYaml = require('js-yaml')
 var walk = require('ignore-walk')
 var execSync = require('child_process').execSync
-var validator = require('validator')
```

Comment:
I'm unsure what benefits `validator` provides. But would it be worthwhile to keep it, and call it in your `sanitizeVar` function before the "&" removal step? Off the cuff, something like:

```
function sanitizeVar(arg) {
  arg = validator.escape(arg)
  return arg.replace(/&/g, '')
}
```

Reply:
Validator was added with the last patch. It also escaped `/` which makes actual paths very unhappy.